### DISCUSSION TITLE
Two questions：1.Where is InstructP2P？2.Why are there two Scribble？

### DISCUSSION THREAD

**Discussion by IDXoX** [Mikubill/sd-webui-controlnet#2797] [Q&A]
Please Help：
1.Where is InstructP2P？
I like it very much,and it's used a lot,but it's gone now,
How do I reinstall it in Controlnet?

2.Why are there two Scribble？

![111](https://github.com/Mikubill/sd-webui-controlnet/assets/165264554/d8cfc25c-7380-4d4b-b03e-6b571c7032a6)

**Comment by huchenlei** [comment]
Fixed in https://github.com/Mikubill/sd-webui-controlnet/pull/2799.
